fix: add min id length admonition (#53)
amplitude/amplitude-dev-center: docs/data/sdks/ios-swift/migration.md

|Customize storage provider. NOT SUPPORTED.|`config.storageProvider`|
|Set up log level. NOT SUPPORTED.|`config.logLevel`|
|Customize logger provider. NOT SUPPORTED.|`config.loggerProvider`|

> |Have no length limit on deviceId or userId.|`config.minIdLength`overwrites the minimum length deviceId and userId.|

Reviewer: ```suggestion
| `deviceId` and `userId` don't have a minimum length.|`config.minIdLength` overwrites the minimum length `deviceId` and `userId`.|
```

@Mercy811 So is the minimum for Amplitude-Swift 5, which you can override with `config.minIdLength`?

Author: Yes!

> | `deviceId` and `userId` don't have a minimum length.|`config.minIdLength` overwrites the minimum length `deviceId` and `userId`.|

Reviewer: ```suggestion
| `deviceId` and `userId` don't have a minimum length.| Minimum length is 5. `config.minIdLength` overwrites the minimum length  of`deviceId` and `userId`.|
```
